x-pack/plugins/fleet/server/routes/epm/handlers.ts

```diff
@@ -317,30 +315,12 @@ export const getBulkAssetsHandler: FleetRequestHandler<
   TypeOf<typeof GetBulkAssetsRequestSchema.body>
 > = async (context, request, response) => {
   try {
-    const savedObjectsClient = (await context.fleet).internalSoClient;
     const { assetIds } = request.body;
+    const savedObjectsClient = (await context.fleet).internalSoClient;
+    const assets = await getBulkAssets(savedObjectsClient, assetIds as AssetSOObject[]);
```

Comment:
I refactored this as I was trying to export it with `PackageService` from `/server/plugin`, so I decided to keep this change even if it's not strictly related to this PR